policyengine_us/variables/gov/states/mi/tax/income/exemptions/mi_exemptions.py

```diff
@@ -0,0 +1,56 @@
+from policyengine_us.model_api import *
+
+
+class mi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan Exemptions"
+    defined_for = StateCode.MI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-30",
+        "https://www.legislature.mi.gov/Publications/TaxpayerGuide.pdf",
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mi.tax.income.exemptions
+
+        # Personal Exemptions
+        person = tax_unit("head_spouse_count", period)
+        personal_exemption = person * p.filer
+
+        # Dependent exemptions
+        dependent = tax_unit("tax_unit_dependents", period)
+        dependent_exemption = dependent * p.dependent
+
+        # Stillborn Exemptions
+        stillborn = tax_unit("tax_unit_stillborn_parent", period)
+        stillborn_exemption = stillborn * p.stillborn
+
+        # Disabled exemptions
+        disabled = tax_unit("head_is_disabled", period)
```

Comment:
doesn't think need to count the number of disabled people in the tax unit? this is only if the head is disabled

Reply:
the leagal code says it's for each taxpayer and every dependent of the taxpayer

>   (3) Except as otherwise provided in subsection (7), a single additional exemption determined as follows shall be subtracted in the calculation that determines taxable income in each of the following circumstances:
  (a) $1,800.00 for each taxpayer and every dependent of the taxpayer who is a deaf person as defined in section 2 of the deaf persons' interpreters act, 1982 PA 204, MCL 393.502; a paraplegic, a quadriplegic, or a hemiplegic; a person who is blind as defined in section 504; or a person who is totally and permanently disabled as defined in section 522. When a dependent of a taxpayer files an annual return under this part, the taxpayer or dependent of the taxpayer, but not both, may claim the additional exemption allowed under this subdivision.

Reply:
I think the law allows for every taxpayer and dependent of the taxpayer 
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/f8158f45-0ec8-46b4-bc3a-7905147e9c16)

Reply:
Right, so you'll want `disabled_people = add(tax_unit, period, ["is_disabled"])`. Please also include a test with multiple disabled members.